WISE-Community/WISE-Client
# refactor(Authoring): Create component to select step and component

## Changes

- Create component to select step and component
- Make all the components that require selecting a step and component use this new component.

## Test

Make sure the authoring works for
- Connected component authoring for components that use the default connected component authoring like Open Response
- Connected component authoring in these specific components that have slightly different connected component authoring
  - Concept Map
  - Draw
  - Graph
  - Label
  - Table
- Edit Dynamic Prompt
- Edit Question Bank
- Create new Peer Grouping
- Edit Peer Grouping

Closes #852
